tests/drivers/build_all/gpio/prj.conf

```diff
@@ -4,3 +4,4 @@ CONFIG_TEST_USERSPACE=y
 CONFIG_I2C=y
 CONFIG_GPIO_PCA95XX_INTERRUPT=y
 CONFIG_SPI=y
+CONFIG_GPIO_ALTERA_PIO=y
```

Comment:
This should not be needed. It will automatically be turned on for the builds using the overlay, Please remove.